src/tacacs/bash/bash_tacplus/bash_tacplus.c

```diff
@@ -0,0 +1,397 @@
+/* bash_tacplus.c - Bash plugin for TACACS+ protocol.
+ * Copyright (C) 2021, Liu Hua <liuh@microsoft.com>
+ *
+ *   TACACS+ work based on pam_tacplus.c
+ *     Copyright (C) 2010, Pawel Krawczyk <pawel.krawczyk@hush.com> and
+ *     Jeroen Nijhof <jeroen@jeroennijhof.nl>
+ *
+ *   TACACS+ authorization work based on tacplus-auth.c
```

Comment:
>TACACS+ authorization work based on tacplus-auth.c [](http://example.com/codeflow?start=5&length=50)

We are using this file to track 3rd party licence. [https://github.com/Azure/sonic-buildimage/blob/master/ThirdPartyLicenses.txt](https://github.com/Azure/sonic-buildimage/blob/master/ThirdPartyLicenses.txt)

Reply:
Fixed, update to the GPL v2 part.